server/src/test/resources/org/opensearch/bootstrap/test.policy

```diff
@@ -7,7 +7,7 @@
  */
 
 grant {
-  // allow to test Security policy and codebases 
+  // allow to test Security policy and codebases
```

Comment:
Leftovers?

Reply:
hmm, I guess there was a trailing space in the comment that my IDE removed.

Reply:
Oh sorry :)